### ISSUE TITLE
Installation fails when executable name doesn’t match package name

### ISSUE THREAD

**Issue by dreymonde** [yonaskolb/Mint#88]
SwiftPM provides a way to customize the name of the final executable by using executables. So, for example, I have a script called `SofImage` which builds as `sofarimage`:

```swift
// swift-tools-version:4.0
// Managed by ice

import PackageDescription

let package = Package(
    name: "SofImage",
    products: [
        .executable(name: "sofarimage", targets: ["SofImage"]),
    ],
    dependencies: [
        .package(url: "https://github.com/onevcat/Rainbow", from: "3.1.4"),
    ],
    targets: [
        .target(name: "SofImage", dependencies: ["Rainbow"]),
    ]
)
```

But **Mint** currently fails when trying to install such package:

```
🌱  Building SofImage with SPM...
Fetching https://github.com/onevcat/Rainbow
Cloning https://github.com/onevcat/Rainbow
Resolving https://github.com/onevcat/Rainbow at 3.1.4
Compile Swift Module 'Rainbow' (11 sources)
Compile Swift Module 'SofImage' (1 sources)
Linking ./.build/x86_64-apple-macosx10.10/release/sofarimage
🌱  Installing...
Couldn't find command SofImage
```

I think that **Mint** should try to link the executable that is specified by SwiftPM if it’s available. If you agree with me, I can look into it and submit a PR. Just let me know!

I totally love Mint, it really can become a very big part of Swift ecosystem, so we should take a proper look at edge cases like this one.

**Comment by dreymonde** [comment]
Oh didn’t see the README, this problem is addressed in some way. Sorry, closing the issue

**Comment by yonaskolb** [comment]
Yeah there’s a workaround, but the proper fix would be #17
